grafana-plugin/src/error-types.ts

```diff
@@ -0,0 +1,56 @@
+const OnCallErrorCode = {
+  // 1 - 999 reserved for engine errors
```

Comment:
https://github.com/grafana/oncall/blob/a865ae1378f0d8399b3e077c5d86593938f99bba/engine/apps/api/errors.py#L3

Reply:
should we use this structure for the codes?

Reply:
Yes, it makes sense